outscale/resource_outscale_net.go

```diff
@@ -71,7 +71,19 @@ func resourceOutscaleOAPINetCreate(d *schema.ResourceData, meta interface{}) err
 
 	d.SetId(resp.Net.GetNetId())
 
-	return resourceOutscaleOAPINetRead(d, meta)
+	return resource.Retry(120*time.Second, func() *resource.RetryError {
+		err = resourceOutscaleOAPINetRead(d, meta)
+		if err != nil {
+			return resource.NonRetryableError(err)
+		}
+		if c, ok := d.GetOk("state"); ok {
+			state := c.(string)
+			if 0 != strings.Compare(state, "available") {
+				return resource.RetryableError(fmt.Errorf("Expected Net to be available but was in state %s", state))
+			}
+		}
+		return resource.RetryableError(err)
```

Comment:
as this seems to be the success return, I think this should be
```
return nil
```

Reply:
Correction done